lib/telephony/pinpoint/sms_sender.rb

```diff
@@ -37,7 +37,7 @@ def send(message:, to:, otp: nil)
                   body: message,
                   message_type: 'TRANSACTIONAL',
                   origination_number: sms_config.shortcode,
-                  sender_id: Telephony.config.sender_id,
+                  sender_id: Telephony.config.country_sender_ids[country_code.to_s],
```

Comment:
**question:** just to make sure I understand this, if this defaults to `nil` Pinpoint just won't use a Sender ID, right? I see that's what the tests say but just confirming.

Reply:
yup!